source/Halibut/ServiceModel/PendingRequestQueueAsync.cs

```diff
@@ -378,5 +378,12 @@ public void Dispose()
                 transferLock?.Dispose();
             }
         }
+
```

Comment:
Nit: I think I'd move this method higher. It feels a little like it's part of `PendingRequest`, where it's not.